From 4b998b1ddf3dc379b7e61dd76a55e3f948b75bd3 Mon Sep 17 00:00:00 2001
From: Stefan Haller <stefan@haller-berlin.de>
Date: Mon, 12 Oct 2020 17:10:02 +0200
Subject: [PATCH 02/20] git-gui/gitk: Delete the code to set an icon

When using our bundled Tcl/Tk 8.6.10, this interferes with the normal display
of the app's icon. Not sure why this wasn't a problem before.
---
 git-gui/git-gui.sh | 25 -------------------------
 gitk-git/gitk      | 23 -----------------------
 2 files changed, 48 deletions(-)

diff --git a/git-gui/git-gui.sh b/git-gui/git-gui.sh
index 201524c34e..3dba934e6e 100755
--- a/git-gui/git-gui.sh
+++ b/git-gui/git-gui.sh
@@ -740,31 +740,6 @@ if {[is_Windows]} {
 	if {![info exists env(DISPLAY)]} {
 		set env(DISPLAY) :9999
 	}
-} else {
-	catch {
-		image create photo gitlogo -width 16 -height 16
-
-		gitlogo put #33CC33 -to  7  0  9  2
-		gitlogo put #33CC33 -to  4  2 12  4
-		gitlogo put #33CC33 -to  7  4  9  6
-		gitlogo put #CC3333 -to  4  6 12  8
-		gitlogo put gray26  -to  4  9  6 10
-		gitlogo put gray26  -to  3 10  6 12
-		gitlogo put gray26  -to  8  9 13 11
-		gitlogo put gray26  -to  8 11 10 12
-		gitlogo put gray26  -to 11 11 13 14
-		gitlogo put gray26  -to  3 12  5 14
-		gitlogo put gray26  -to  5 13
-		gitlogo put gray26  -to 10 13
-		gitlogo put gray26  -to  4 14 12 15
-		gitlogo put gray26  -to  5 15 11 16
-		gitlogo redither
-
-		image create photo gitlogo32 -width 32 -height 32
-		gitlogo32 copy gitlogo -zoom 2 2
-
-		wm iconphoto . -default gitlogo gitlogo32
-	}
 }
 
 ######################################################################
diff --git a/gitk-git/gitk b/gitk-git/gitk
index cf70313dd8..7644a12697 100755
--- a/gitk-git/gitk
+++ b/gitk-git/gitk
@@ -12659,29 +12659,6 @@ if {[expr {[exec git rev-parse --is-inside-work-tree] == "true"}]} {
 set worktree [gitworktree]
 setcoords
 makewindow
-catch {
-    image create photo gitlogo      -width 16 -height 16
-
-    image create photo gitlogominus -width  4 -height  2
-    gitlogominus put #C00000 -to 0 0 4 2
-    gitlogo copy gitlogominus -to  1 5
-    gitlogo copy gitlogominus -to  6 5
-    gitlogo copy gitlogominus -to 11 5
-    image delete gitlogominus
-
-    image create photo gitlogoplus  -width  4 -height  4
-    gitlogoplus  put #008000 -to 1 0 3 4
-    gitlogoplus  put #008000 -to 0 1 4 3
-    gitlogo copy gitlogoplus  -to  1 9
-    gitlogo copy gitlogoplus  -to  6 9
-    gitlogo copy gitlogoplus  -to 11 9
-    image delete gitlogoplus
-
-    image create photo gitlogo32    -width 32 -height 32
-    gitlogo32 copy gitlogo -zoom 2 2
-
-    wm iconphoto . -default gitlogo gitlogo32
-}
 # wait for the window to become visible
 tkwait visibility .
 set_window_title
-- 
2.35.0.20.gda5c78a719

